src/invidious/helpers/youtube_api.cr

```diff
@@ -67,6 +67,48 @@ def request_youtube_api_browse(browse_id : String, params : String, region : Str
   return _youtube_api_post_json("/youtubei/v1/browse", data)
 end
 
+####################################################################
+# request_youtube_api_next(continuation)
+# request_youtube_api_next(video_id, params)
+#
+# Requests the youtubei/v1/next endpoint with the required headers
+# and POST data in order to get a JSON reply in english that can
+# be easily parsed.
+#
+# The requested data can either be:
+#
+#  - A continuation token (ctoken). Depending on this token's
+#    contents, the returned data can be comments, playlist videos,
```

Comment:
could you check what data is available in there, and adapt the comments accordingly? (from what I know, video comments and their replies are returned, right?)

Reply:
For the moment, I only know that video comments and the replies can be returned. I'll adapt the code for that.